BLSQ/iaso
# POLIO-1276 [Line-list] Polio notifications front-end

Front-end for Polio notifications.

Related JIRA tickets : [POLIO-1309](https://bluesquare.atlassian.net/browse/POLIO-1276)

## Changes

Front-end for [Polio notifications API](https://github.com/BLSQ/iaso/pull/928).

New UIs for notifications:

- CRUD
- bulk update

## Related PR (`bluesquare-components`)

https://github.com/BLSQ/bluesquare-components/pull/135/files

## How to test

- ensure to be admin or to have the right perms (`POLIO_NOTIFICATIONS`)
- create notifications:
    1. either manually
    1. or by importing an .xlsx file ([sample available here](https://bluesquare.atlassian.net/browse/POLIO-1275?focusedCommentId=59457))
        - the backend's import is async, so a worker must be running
        - `docker-compose run iaso manage tasks_worker`
- edit notifications
- delete notifications

## Print screen / video


https://github.com/BLSQ/iaso/assets/281139/91b6b929-dd58-4d8e-af3e-604dfc71e5bc


[POLIO-1309]: https://bluesquare.atlassian.net/browse/POLIO-1309?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
